cmd/mount.go

```diff
@@ -561,16 +578,23 @@ var mountCmd = &cobra.Command{
 					buff, err := os.ReadFile(dmnCtx.LogFileName)
 					if err != nil {
 						log.Err("mount: failed to read child [%v] failure logs [%s]", child.Pid, err.Error())
-						return Destroy(fmt.Sprintf("failed to mount, please check logs [%s]", err.Error()))
+						err = fmt.Errorf("failed to mount, please check logs [%s]", err.Error())
 					} else {
-						return Destroy(string(buff))
+						err = fmt.Errorf(string(buff))
+					}
+
+					// Safe to delete the temp file.
+					rmErr := os.Remove(childStdErrFileName)
+					if rmErr != nil {
```

Comment:
After starting the child process, parent will go down and delete this file. Ideally this file shall be created by child and deleted only when child exits. Only in case of foreground process may be parent can also have logic to use this file.